spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsTransmitter.java

```diff
@@ -65,6 +65,9 @@
     @Value("${ygg.admin.api-base-url}")
     private String adminBaseUrl;
 
+    @Value("${ygg.admin.server.url}")
```

Comment:
Should the parameter also be removed here, as you removed it on the GitService class?

Reply:
Done. Already removed.